Peoplesoft/main.tf

```diff
@@ -0,0 +1,538 @@
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["bastion"]}"                
+            destination_port_ranges = "22" 
+            destination_address_prefix = "*"             
+        },
+               {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["elasticsearch"]}"                
+            destination_port_ranges = "*" 
+            destination_address_prefix = "*"             
+        }
+    ]
+
+    webserver_sr_outbound = [
+                {
+            source_port_ranges =  "*" 
+            source_address_prefix = "*"  # ob to Application Servers               
+            destination_port_ranges = "*" 
+            destination_address_prefix = "${local.subnetPrefixes["application"]}"             
+        },
+                   {
+            source_port_ranges =  "*" 
+            source_address_prefix = "*"              
+            destination_port_ranges = "9200" 
+            destination_address_prefix = "${local.subnetPrefixes["elasticsearch"]}"    # ob to Elastic Servers            
+        }
+    ]
+    elasticsearch_sr_inbound = [
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["webserver"]}"             
+            destination_port_ranges = "9200"
+            destination_address_prefix = "*"             
+        },
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["bastion"]}"                
+            destination_port_ranges = "22" 
+            destination_address_prefix = "*"             
+        }
+    ]
+
+    elasticsearch_sr_outbound = [
+                {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["application"]}"  # ob to Application Servers               
+            destination_port_ranges = "9033-9039" 
+            destination_address_prefix = "*"             # Need to support ASG
+        },
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["webserver"]}"  # ob to WebServers            
+            destination_port_ranges = "8000" 
+            destination_address_prefix = "*"            
+        }
+    ]
+
+        toolsclient_sr_inbound = [
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["application"]}"             
+            destination_port_ranges = "5985-5986"
+            destination_address_prefix = "*"             
+        },
+        {
+            source_port_ranges =  "*" 
+            source_address_prefix = "${local.subnetPrefixes["bastion"]}"                
+            destination_port_ranges = "22" 
+            destination_address_prefix = "*"             
+        }
+    ]
+
+    toolsclient_sr_outbound = [
+
+    ]
+    
+    database_sr_inbound = [
```

Comment:
don't need database NSG. Comment for now. Don't remove it.

Reply:
done